README.md

```diff
@@ -185,3 +185,7 @@ Multiple example apps are included:
 - the `counter` application, which illustrates nonce checking in txs
 - the `dummy` application, which illustrates a simple key-value merkle tree
 - the `dummy --persistent` application, which augments the dummy with persistence and validator set changes
+
+When you are developing an implementation of the ABCI server in your favourite language, we provide the `abci-cli test`
+command that allows you to run our tests against your own ABCI server. Please make sure that your server supports both
+gRPC and the TSP socket protocol.
```

Comment:
Agreed! We only really need TSP. gRPC is just a convenience and not a requirement, especially since its not even supported yet in every language :)